main.go

```diff
@@ -7,13 +7,18 @@ import (
 
 	"github.com/manifoldco/torus-cli/cmd"
 	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/prefs"
+	"github.com/manifoldco/torus-cli/ui"
 )
 
 func main() {
 	cli.VersionPrinter = func(ctx *cli.Context) {
 		cmd.VersionLookup(ctx)
 	}
 
+	preferences, _ := prefs.NewPreferences(true)
```

Comment:
This goes back to displaying *all* prefs with a prefs list, and setting defaults into the file on a prefs set.

I guess that's better than having to do the negated option values though